.dockerignore

```diff
@@ -1,2 +1,10 @@
-Dockerfile
-Dockerfile.dev
+/.git/
+/.github/
+/.gitmodules/
```

Comment:
When ever copying something to image, this will exclude these files & folders from context. Meaning they will not be copied AND not be part of the context.